src/components/Page/components/Header/Header.tsx

```diff
@@ -2,23 +2,37 @@ import React from 'react';
 import {classNames} from '../../../../utilities/css';
 import {buttonsFrom} from '../../../Button';
 import {useMediaQuery} from '../../../../utilities/media-query';
-import {ComplexAction, MenuGroupDescriptor} from '../../../../types';
+import {
+  ComplexAction,
+  MenuGroupDescriptor,
+  AppBridgeAction,
+  DestructableAction,
+  DisableableAction,
+  LoadableAction,
+} from '../../../../types';
 import {Breadcrumbs, BreadcrumbsProps} from '../../../Breadcrumbs';
-
 import {Pagination, PaginationDescriptor} from '../../../Pagination';
 import {ActionMenu, hasGroupsWithActions} from '../../../ActionMenu';
 
-import {HeaderPrimaryAction} from '../../types';
 import {Title, TitleProps} from './components';
 import styles from './Header.scss';
 
+interface PrimaryAction
```

Comment:
This interface previously lived in the types file but it was only used here, so just inline it